src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx

```diff
@@ -403,6 +412,7 @@ export default class CreateMarketOutcome extends Component {
             onChange={(e) => {
               updateNewMarket({ detailsText: e.target.value })
             }}
+            onKeyPress={e => keyPressed(e)}
           />
```

Comment:
Let's not capture keyPressed for additional details. The use will probably want to format the text which will include carriage returns. 

They'll have to tab out of the textarea in order to hit <enter> and advance.